android/app/src/main/java/com/zulipmobile/notifications/FcmMessage.kt

```diff
@@ -112,6 +112,7 @@ data class MessageFcmMessage(
     fun dataForOpen(): Bundle = Bundle().apply {
         // NOTE: Keep the JS-side type definition in sync with this code.
         identity.realmUri.let { putString("realm_uri", it.toString()) }
+        identity.userId.let { putInt("user_id", it) }
```

Comment:
The `let` here is redundant; can simplify it out.

(It's redundant on the previous line too. Looks like it got left behind in 9620629a7571c7d981c54384cfe9f3d72c9fadbc when I made `realmUri` non-nullable; before that it was `?.let` and was doing something useful.)